packages/web/src/pages/pay-and-earn-page/desktop/PayAndEarnPage.tsx

```diff
@@ -139,7 +139,13 @@ export const PayAndEarnPage = ({ tableView }: PayAndEarnPageProps) => {
         <LoadingSpinner className={styles.spinner} />
       ) : (
         <>
-          <USDCCard balance={balance} />
+          <USDCCard
+            balance={balance}
+            refreshing={
+              balanceStatus === Status.LOADING ||
```

Comment:
`StatusIsNotFinalized`? maybe overkill

Reply:
Given that this is intended to show a spinner next to the balance while a fetch is in progress, I would like it to specifically only show while we're in the loading state. And we definitely don't want it to show while recovery is in `IDLE` state.